pandapower/create.py

```diff
@@ -3886,6 +3886,68 @@ def create_pwl_cost(net, element, et, points, power_type="p", index=None, check=
     return index
 
 
+def create_pwl_costs(net, elements, et, points, power_type="p", index=None, check=True, **kwargs):
+    """
+    Creates an entry for piecewise linear costs for an element. The currently supported elements are
```

Comment:
more precisely: Creates entries for piecewise linear costs for multiple elements of the same type.